src/components/App.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import Page from './Page';
+
+function App() {
+  const [state, setState] = useState({
+    count: 0,
+  });
+
+  const { count } = state;
```

Comment:
```suggestion
  const [count, setCount] = useState(0);
```

상태가 추가적으로 있는게 아니라면 간단하게 표현해줘도 괜찮을 것 같아요.

Reply:
앗 줄여보도록 하겠습니다!!